Gaijinizer patch/Map059.txt

```diff
@@ -66,11 +69,10 @@ at the shelter.#MTLed
 
 んで、監視カメラ覗いたら、お前さんがおった。
 > CONTEXT: Map059/events/1/pages/0/57/Dialogue/other3
-In an underground shelter in this area where
-evacuation should have been completed
-Suddenly the power in the medical area was fully
-open and I thought it was suspicious.
-Well, when I saw a surveillance camera, you came.#MTLed
+That shelter was supposed to have been fully
+evacuated and then all of a sudden the power in
+the medical unit came on, so I got suspicious. And
+when I peeked through the camera, there you were.#Ron
```

Comment:
Minor flow:
>_**That shelter was supposed to have been fully
evacuated, and then all of a sudden the power in
the medical unit came on.**_
>
>_**I got suspicious, so I started looking
through the camera footage... 
and there you were.**_